x-pack/plugins/transform/common/types/data_view.ts

```diff
@@ -5,18 +5,18 @@
  * 2.0.
  */
 
-import type { IndexPattern } from '../../../../../src/plugins/data/common';
+import { DataView } from '../../../../../src/plugins/data_views/common';
```

Comment:
Can we keep the `type` part here?

Reply:
Done in https://github.com/elastic/kibana/pull/128247/commits/b846d838066b7befa01f487a1d1caa88bcfeffbd